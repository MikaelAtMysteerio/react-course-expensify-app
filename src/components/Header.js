import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expensify App</h1>
        <NavLink to="/" activeClassName="is-active" exact={true}>Dashboard </NavLink>
        <NavLink to="/AddPage" activeClassName="is-active" >AddPage </NavLink>
    </header>
);

export default Header;
