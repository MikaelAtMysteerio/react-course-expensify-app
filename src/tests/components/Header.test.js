import React from 'react';
import {shallow} from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Header from '../../components/Header';

Enzyme.configure({
    adapter: new Adapter()
});

test('test header component', () => {
    const wrapper = shallow(<Header/>);
    expect(wrapper).toMatchSnapshot;
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot()
});