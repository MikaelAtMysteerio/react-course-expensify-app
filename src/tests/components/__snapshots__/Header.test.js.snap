// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test header component 1`] = `
<header>
  <h1>
    Expensify App
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/AddPage"
  >
    AddPage 
  </NavLink>
</header>
`;
